src/storage/statistics_cl.c

```diff
@@ -337,6 +344,9 @@ stats_dump (const char *class_name_p, FILE * file_p)
       name_p = sm_get_att_name (class_mop, attr_stats_p->id);
       fprintf (file_p, " Attribute: %s\n", (name_p ? name_p : "not found"));
       fprintf (file_p, "    id: %d\n", attr_stats_p->id);
+      /*
+         fprintf (file_p, "    ndv: %ld\n", attr_stats_p->ndv);
+       */
```

Comment:
If there is no reason to leave it as a comment, please clear it.

Reply:
it's related to next PR. I keep it for next PR.